ouroboros-network/sim-tests-lib/Test/Ouroboros/Network/PeerSelection/RootPeersDNS.hs

```diff
@@ -50,6 +50,18 @@ import           Control.Tracer (Tracer (Tracer), contramap)
 import           Data.List (intercalate)
 import           Ouroboros.Network.PeerSelection.PeerAdvertise
                      (PeerAdvertise (..))
+import           Ouroboros.Network.PeerSelection.RelayAccessPoint
+                     (DomainAccessPoint (..), RelayAccessPoint (..))
+import           Ouroboros.Network.PeerSelection.RootPeersDNS.DNSActions
+                     (DNSActions (..), constantResource)
+import           Ouroboros.Network.PeerSelection.RootPeersDNS.DNSSemaphore
+                     (newLedgerAndPublicRootDNSSemaphore)
+import           Ouroboros.Network.PeerSelection.RootPeersDNS.LedgerPeers
+                     (TraceLedgerPeers, resolveLedgerPeers)
+import           Ouroboros.Network.PeerSelection.RootPeersDNS.LocalRootPeers
+                     (TraceLocalRootPeers (..), localRootPeersProvider)
+import           Ouroboros.Network.PeerSelection.RootPeersDNS.PublicRootPeers
+                     (TracePublicRootPeers (..), publicRootPeersProvider)
```

Comment:
It's better to have unrestricted imports in tests, it makes pasting counterexamples much easier.